install_M1.sh

```diff
@@ -0,0 +1,35 @@
+# /bin/sh
+
+if [[ $(uname) != 'Darwin' ]]; then
+    printf "\e[1;31mThis is for MacOS only!\e[0m\n"
+    exit 1
+fi
+
+if [[ $(uname -m) != 'arm64' ]]; then
+  printf "\e[1;31mThis script is for M1 chips only!\e[0m\n"
```

Comment:
Instead of specifying M1 chips, you should use a more generic term like “AppleSilicon chip’ as today there are M1, M2, and in a close future M3 chips :)

Reply:
Totally true, will fix it! Thanks